src/promptflow-tracing/tests/e2etests/test_tracing.py

```diff
@@ -222,6 +223,22 @@ def assert_otel_traces_with_embedding(self, dev_connections, func, inputs, expec
     def test_otel_trace_with_multiple_functions(self):
         execute_function_in_subprocess(self.assert_otel_traces_with_multiple_functions)
 
+    def test_otel_tracer_refreshed_after_start_trace(self):
+        from promptflow.tracing import start_trace
+
+        memory_exporter = prepare_memory_exporter()
```

Comment:
Should we use subprocess to run the test to avoid side effect?

Reply:
do other test use subprocess? i just copy lines, adding `start_trace`.

Reply:
You can see the test_tracing, which is using a function execute_function_in_subprocess

Reply:
I see, let me update this